scripts/backup.js

```diff
@@ -4,17 +4,22 @@ const argv = require('minimist')(process.argv.slice(2));
 const { exec } = require('child_process');
 require('dotenv').config();
 
+let environment;
 if (!process.env.NODE_ENV) {
     // Environment not set. Use the production.
-    process.env.NODE_ENV = 'testnet';
-} else if (['development', 'testnet', 'mainnet'].indexOf(process.env.NODE_ENV) < 0) {
+    environment = 'testnet';
+} else if (['development', 'testnet', 'mainnet', 'mariner'].indexOf(process.env.NODE_ENV) < 0) {
```

Comment:
Why do we use mariner here?

Reply:
For backward compatibility purposes... Nodes created before the 4.0.0 release have mariner as their NODE_ENV so that needed to be taken into account